_stbt/core.py

```diff
@@ -1668,6 +1671,25 @@ def draw(self, obj, duration_secs=None, label=""):
                     "Can't draw object of type '%s'" % type(obj).__name__)
 
 
+class NoSinkPipeline(object):
+    """
+    Used in place of a SinkPipeline when no video output is required.  Is a lot
+    faster because it doesn't do anything.  It especially doesn't do any copying
+    nor video encoding :).
```

Comment:
We don't do video encoding either if `--save-video` isn't set. Do you have a rough indication of what the performance gain is? What copying are we doing without this -- I assume the only copy is in `gst_sample_make_writable` called from `push_sample`.